third_party/amd/lib/TritonAMDGPUToLLVM/BufferOpsEmitter.h

```diff
@@ -67,14 +67,16 @@ struct BufferEmitter {
 
   // Emit a predicated rocdl.raw.ptr.buffer.load
   Value emitLoad(Type type, Value rsrcDesc, Value offset, Value pred,
-                 Value falseVal);
+                 Value falseVal, CacheModifier cm);
```

Comment:
Ah, I see here we're targeting rocdl.raw.ptr.buffer.load (https://mlir.llvm.org/docs/Dialects/ROCDLDialect/#rocdlrawptrbufferload-rocdlrawptrbufferloadop).

@ramjana You had created a table for comparing PTX to AMD cache modifiers. Do we have that for earlier architectures, and will we always be able to create a 1:1 mapping for future architectures?

Reply:
I did refer to Raman's table while coding the cache modifiers. It'd be great to have a similar comparison for other architectures.